packages/game/src/reducers/statsReducer.ts

```diff
@@ -216,8 +217,17 @@ function statsReducerFunction(
   }
 }
 
-function isBlindPlay(action: GameAction, gameState: GameState): boolean {
-  if (action.type !== "play") {
+function isBlindPlay(
+  action: GameAction,
+  gameState: GameState,
+  variant: Variant,
+): boolean {
+  // In "Throw it in a Hole" variants, bombs should appear as successful plays.
+  const possiblePlay =
+    action.type == "play" ||
+    (variant.throwItInAHole && action.type == "discard" && action.failed);
```

Comment:
whoops, sorry, I botched the `===` on these. Is this easy for you to fix?

Reply:
you can just fix it yourself by pushing another commit to the fork

Reply:
ack. is there a way for me to run the lints myself before the website does it? to figure out what's broken?

Reply:
yeah just run lint.sh

Reply:
but you should see the errors in your editor (vscode) immediately as you are coding, otherwise you are going to be having a real shitty time, dm me on discord and i can help you set it up

Reply:
thanks for the help over discord! also fixed in the latest commit.